daml-lf/transaction/src/main/protobuf/com/daml/lf/transaction.proto

```diff
@@ -69,18 +61,18 @@ message NodeCreate {
 }
 
 message NodeFetch {
-    string contract_id = 1;
+    reserved 1; // was contract_id
     com.daml.lf.value.Identifier template_id = 2;
     repeated string actors = 7;
     repeated string stakeholders = 3;
     repeated string signatories = 4;
-    string value_version = 5;
+    reserved 5; // was value_version
```

Comment:
Is always "6" since SDK 1.0.0.

Reply:
The value is check (that is actually undefined or set to a string between "1" to "6"), but the field had no actual semantics impact.

Reply:
it is always unset since #6784

Reply:
Did we only have a value version but no value itself?

Reply:
There is no value in Fetch node. 
It was conceptually there to version the template ID (template ID leave in the value.proto file)
However the format of template ID has never changed, so it had never any impact.

Reply:
Current specification authorize the field to be one of the strings "1", "2", "3", "4", "5", "6".
Do you think it is worthwhile to check it ?

Reply:
Would one come from pre-1.0.0 SDKs?

Reply:
Unfortunate no. It can be produced by SDK 1.3.0 or older.

Reply:
Does the field carry any meaning under the current implementation? If not, it's probably fine to ignore it.

Reply:
I wrote it above
>  the field had no actual semantics impact.
Thanks.